### ISSUE TITLE
Overriding pnpm linked deps

### ISSUE THREAD

**Issue by vjpr** [pnpm/pnpm#819]
Currently if you have linked a dep and want to override it you have to do the following:

```
pnpm link foo
pnpm unlink foo # or rm node_modules/foo
pnpm i foo
```

Now if you have 10 linked deps and you want to unlink them all (e.g. to test that your app works without links), it can be time-consuming.

- [x] Allow linking multiple packages at once `pnpm link foo bar baz`.
- [ ] Perhaps `pnpm unlink` should install the dep that was unlinked instead of just being an alias for `pnpm uninstall` (#298), or there should be a flag to install the dep.

**Comment by zkochan** [comment]
Although this is not how `npm unlink` works, it seems logical to me. So I agree.

But we'll probably have to print some warnings or notice to inform people that it works not the way it does with npm.

**Comment by zkochan** [comment]
but this would be a breaking change and I would wait a while with a new major version, so for now we can use a new command. Maybe `pnpm dislink`

**Comment by zkochan** [comment]
`pnpm dislink` published with [v1.6.0](https://github.com/pnpm/pnpm/releases/tag/v1.6.0).

When called w/o arguments, all external links are removed. If the linked package is in `package.json`, it is reinstall after the link is removed
